Project1/Program.cs

```diff
@@ -1 +1,56 @@
-﻿
\ No newline at end of file
+﻿using Project1;
+
+internal class Program
+{
+    private static void Main(string[] args)
+    {
+        Menu menu = new Menu();
+        menu.ShowMainMenu();
+        Console.Write("\nWybierz numer z manu głównego: ");
+        int numberMainMenu = Convert.ToInt32(Console.ReadLine());
+
+        int value1;
+        int value2;
+        if (numberMainMenu == 1)
+        {
+            Console.WriteLine();
+            CalculatorBase calculatorbase = new CalculatorBase();
+            menu.ShowBaseCalculatorMenu();
+            Console.Write("\nWybierz numer z menu prostego kalkulatora: ");
```

Comment:
tak samo jak wyżej czemu ten \n i to tyczy się juz wszytskiego co niżej też to będzie miało